src/rfac/rfac.c

```
@@ -0,0 +1,277 @@
+
+/*!
+ * Performs a deep copy of an #rfac_iv structure.
+ *
+ * \param[out] dest Pointer to copied structure.
+ * \param[in] src Pointer to source structure.
+ *
+ * \warning Nothing will be done if \p src is \c NULL .
+ */
+void rfac_iv_copy(rfac_iv *dest, const rfac_iv *src)
+{
+  if (src == NULL) return; /* nothing to do */
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)